api/v1/sriovnetworknodestate_types.go

```diff
@@ -64,6 +64,7 @@ type InterfaceExt struct {
 	NumVfs            int               `json:"numVfs,omitempty"`
 	LinkSpeed         string            `json:"linkSpeed,omitempty"`
 	LinkType          string            `json:"linkType,omitempty"`
+	LinkStatus        string            `json:"linkStatus,omitempty"`
```

Comment:
I don't think we need to track the PF linkStatus

Reply:
This is based on the above comment right? If we converge on the comment above, I will remove this change.

Reply:
I think we have two different stuff here
1. Operational link state
2. Actual link state (if the card is not connected to anything for example)

if you ask me I will be interesting in see the actual link state on the nodeState the Opr one can always be up I think as we have today right?